mgmt_cli_test.py

```diff
@@ -96,11 +96,62 @@ def test_manager_sanity(self):
         4) test_client_encryption
         :return:
         """
+        self.test_basic_backup()
         self.test_repair_multiple_keyspace_types()
         self.test_mgmt_cluster_crud()
         self.test_mgmt_cluster_healthcheck()
         self.test_client_encryption()
 
+    def update_cred_file(self):
+        # FIXME: add to the nodes not in the same region as the bucket the bucket's region
+        # this is a temporary fix, after https://github.com/scylladb/mermaid/issues/1456 is fixed, this is not necessary
+        cred_file = '/etc/scylla-manager-agent/scylla-manager-agent.yaml'
+        region = self.params.get('region_name').split(' ')[0]
```

Comment:
I think you are better with hard-coding the region of `manager-backup-tests-us` bucket.
even changing the order of regions, would break this function

Reply:
updated